cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -154,3 +170,28 @@
   @saved_results = @saved_results || {}
   @saved_results[name] = @result
 end
+
+# TODO: is it right place?  This is ugly right now...
+# TODO: the host factory and other concern need to be split apart
+Then("our JSON should be:") do |json|
+  @result.delete('created_at') # TODO: why are we doing this?
+  if @response_api_key
+    json = json.gsub("@response_api_key@", @response_api_key)
+  end
+  json = render_hf(json)
+  expect(@result).to eq(JSON.parse(json))
+end
+
+# TODO: we need a better refactoring for this
+Then("the host factory JSON should be:") do |json|
+  @result.delete('created_at') # TODO: why are we doing this?
```

Comment:
This todo doesn't make sense if you added this code. If it's copy/paste it's because we want to compare static expectation text probably and don't want to deal with dynamically generated `created_at`.

Reply:
Yes, copy paste.  I think I meant was meant in the spirit of "there's something deeper wrong if we're doing this."  

This test code in general -- the cucumber steps and helper modules -- is perhaps the messiest in the codebase.  The TODO isn't very helpful it's more of a marker that something is off.

But the one above accomplishes that so I'll just remove it.